### PR TITLE
Ctx 1085 move to before action

### PR DESCRIPTION
**Pull request by Natay**
Trying to update Banyan to 5.2 is causing following issue here 

```
NoMethodError:
  undefined method `before_filter' for ApplicationController:Class
  Did you mean?  before_action
```

### CODE REVIEW

**Review by nguilford — COMMENTED** [review]
You will also need to update the version in `lib/grnds/sso/version.rb` and then run bundle install to update the Gemfile. It looks like this is a backwards incompatible change, depending on the version of ruby the consumer is using, so you should update the major version.

Looks like the build is failing because of a bundler/ruby version mismatch as well. Tracking that in https://includedhealth.atlassian.net/browse/DP-2052. Made it high priority for DP to dig into next week.

**Comment by nguilford** [comment]
> Should probably also bump the version in the gemspec, so you can run the old version of the gem in older versions of rails. :)

gemspec pulls from version number from `Grnds::Sso::VERSION`.

**Comment by Natay** [comment]
Thanks both for the reviews! I added the suggested changes and attempted to fix bundler/ruby version mismatch in config.yaml, let me know if it's better handled in another PR by DP and I can remove it.

File: lib/grnds/sso/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module Grnds
   module Sso
-    VERSION = '2.4.0'.freeze
+    VERSION = '2.4.1'.freeze
```
Comment (nguilford) [inline lib/grnds/sso/version.rb:3]: Can we make this version 3.0.0 since this is a backwards incompatible change for some versions of rails?

Comment (Natay) [reply lib/grnds/sso/version.rb]: Sorry for the delay! I missed this comment in my emails, got it updated.

**Review by nguilford — APPROVED** [review]

**Pull request merged**